src/index.ts

```diff
@@ -58,55 +57,53 @@ export async function createWebdriverIO(opts: ProgramOpts) {
     const root = path.resolve(process.cwd(), projectDir || '')
 
     /**
-     * check if a package.json exists and if not create one
+     * find package manager that was used to create project
      */
-    const project = await readPackageUp({ cwd: root })
-    if (!project) {
+    const pm = PMs.find((pm) => (
+        // for pnpm check for "~/Library/pnpm/store/v3/..."
+        process.argv[1].includes(`${path.sep}${pm}${path.sep}`) ||
+        // for NPM and Yarn check for "~/.npm/npx/..." or "~/.yarn/bin/create-wdio"
+        process.argv[1].includes(`${path.sep}.${pm}${path.sep}`)
+    )) || 'npm'
+
+    const hasPackageJson = await fs.access(path.resolve(root, 'package.json')).then(() => true).catch(() => false)
```

Comment:
We should use `read-pkg-up` for the use case that someone is running `npm init wdio@latest .` in a sub directory of a project

Reply:
This logic should already be covered, can you explain what expectation you have?

Reply:
Given the project root is `/foo` and I have a `package.json` in `/foo/package.json` and I call `npm init wdio@latest .` in `/foo/bar` then I expect that this package runs the setup from the root of my project (`/foo`) and __not__ from the working directory (`/foo/bar`).